Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/core/model/CoronaTest.kt

```diff
@@ -63,7 +63,7 @@ data class CoronaTest(
     }
 
     fun getFormattedRegistrationDate(): String =
-        registeredAt.toUserTimeZone().toLocalDate().format(DateTimeFormatter.ofLocalizedDate(FormatStyle.SHORT))
+        registeredAt.toLocalDateTimeUserTz().toLocalDate().format(DateTimeFormatter.ofLocalizedDate(FormatStyle.SHORT))
```

Comment:
Wouldn't it be better to have a `toUserTimeZone` extension function like we had before in the `TimeAndDateExtensions`? I see we already have a `JavaTimeAndDateExtensions` file.

Reply:
https://github.com/corona-warn-app/cwa-app-android/pull/5593#discussion_r982141284